omod/src/main/java/org/openmrs/module/registrationapp/page/controller/EditSectionPageController.java

```diff
@@ -237,10 +237,8 @@ private void updatePatientRelationships(Patient patient, String[] types, String[
 
                     // Last character reveals relationship direction (aIsToB or bIsToA)
                     char relationshipDirection = types[i].charAt(types[i].length() - 1);
-                    if (relationshipDirection != 'A') {
-                        if (relationshipDirection != 'B') {
+                    if (relationshipDirection != 'A' && relationshipDirection != 'B') {
```

Comment:
Spaces? Are you sure you shouldn't use tabs?

Reply:
> Spaces? Are you sure you shouldn't use tabs?

This seems obvious but have not figured out with what you mean by spaces Vs tabs. @sacull could you give a hint on this

Reply:
> > Spaces? Are you sure you shouldn't use tabs?
> 
> This seems obvious but have not figured out with what you mean by spaces Vs tabs. @sacull could you give a hint on this

@kdaud,@sacull is trying to say try to organize your code ie dont pull the if statement to the far left,check on the existing if statements in  the file on the way they are organized